testsuite/features/support/namespaces/system.rb

```diff
@@ -57,18 +81,58 @@ def bootstrap_system(host, activation_key, salt_ssh)
     end
   end
 
+  ##
+  # `schedule_apply_highstate` schedules a highstate to be applied to a server at a given date
```

Comment:
```suggestion
  # `schedule_apply_highstate` schedules a highstate to be applied to a server at a given date
```
Sometimes you use the sentence strucutre
- This schedules....
- It schedules...
- `function_name` schedules

I think we should be more consistent.

Reply:
Yup, make sense, as said I only automatically generated them.